src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -1267,6 +1268,66 @@ internal static void TagActivityWithOrchestrationStatus(OrchestrationRuntimeStat
                 activity.AddTag("DurableFunctionsRuntimeStatus", statusStr);
             }
         }
+
+        internal IScaleMonitor GetScaleMonitor(string functionId, FunctionName functionName, string storageConnectionString)
+        {
+            if (this.defaultDurabilityProvider is AzureStorageDurabilityProvider)
+            {
+                // use the Azure Storage scale monitor
+                return new DurableTaskScaleMonitor(
+                        functionId,
+                        functionName,
+                        this.Options.HubName,
+                        storageConnectionString,
+                        this.TraceHelper);
+            }
+            else if ((this.defaultDurabilityProvider is ProviderUtils.IProviderWithAutoScaling provider)
+                     && provider.TryGetScaleMonitor(this, out var scaleMonitor))
+            {
+                // use the scale monitor given by the provider
+                return scaleMonitor;
+            }
+            else
+            {
+                // the durability provider does not support runtime scaling.
+                // Create an empty scale monitor to avoid exceptions (unless runtime scaling is actually turned on).
+                return new PseudoScaleMonitor($"{functionId}-DurableTaskTrigger-{this.Options.HubName}".ToLower());
+            }
+        }
+
+        /// <summary>
+        /// A placeholder scale monitor, can be used by durability providers that do not support runtime scaling.
+        /// This is required to allow operation of those providers even if runtime scaling is turned off
+        /// see discussion https://github.com/Azure/azure-functions-durable-extension/pull/1009/files#r341767018.
+        /// </summary>
+        private sealed class PseudoScaleMonitor : IScaleMonitor

```

Comment:
nit: I'd avoid calling it a "pseudo" scale monitor - my first impression is that it's something for testing. Maybe `NullScaleMonitor` or `NoOpScaleMonitor`?